articles/log-analytics/log-analytics-azure-networking-analytics.md

```diff
@@ -19,15 +19,27 @@ ms.author: richrund
 ---
 # Azure networking monitoring solutions in Log Analytics
 
-You can use the Azure Application Gateway analytics solution in Log Analytics to review:
-
-* Azure Application Gateway logs
-* Azure Application Gateway metrics
-
-You can use the Azure Network Security Group analytics solution in Log Analytics to review:
-
-* Azure Network Security Group logs
-
+Microsoft offers the following solutions for monitoring your networks:
+* Network Performance Monitor (NPM)
+ * Monitor the health of your network
+* Azure Application Gateway analytics to review
+ * Azure Application Gateway logs
+ * Azure Application Gateway metrics
+* Azure Network Security Group analytics to review
+ * Azure Network Security Group logs
+
+All of the above solutions are based on Log Analytics.   
+
+## Network Performance Monitor (NPM)
+NPM is a network monitoring solution, that monitors the health, availability and reachability of networks.  It is used to monitor connectivity between:
+* public cloud and on-premises 
+* data centers and user locations (branch offices)
+* subnets hosting various tiers of a multi-tiered application.
+
+ ![image of Network Performance Monitor](https://docs.microsoft.com/en-us/azure/log-analytics/media/log-analytics-network-performance-monitor/npm-topology.png)
+More information on NPM is available [online](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-network-performance-monitor).  [Announcement](https://blogs.technet.microsoft.com/msoms/2017/02/27/oms-network-performance-monitor-is-now-generally-available/)
```

Comment:
Sorry - this got missed off the review - remove this and updated Line 34 with the link to the NPM solution (using a relative link):

`The [Network Performance Monitoring Solution](log-analytics-network-performance-monitor.md) is a network monitoring solution...`
and omit the reference to the blog post.